python/mxnet/gluon/block.py

```diff
@@ -1137,7 +1141,11 @@ def register_child(self, block, name=None):
         super(HybridBlock, self).register_child(block, name)
         self._clear_cached_op()
 
-    def hybridize(self, active=True, backend=None, backend_opts=None, clear=True, **kwargs):
+    def hybridize(self, active=True, backend=None, clear=True,
```

Comment:
we still need kwargs for hybridize, right?

Reply:
this PR formalizes the args that get passed to CachedOp as parameters to hybridize. Theres no need for kwargs anymore.

Reply:
How would users pass backend_opts with hybridize now?

Reply:
They have to pass them through `optimize_for`

Reply:
Given the latest decision, we're going to make all CachedOp options explicit and all kwargs will go to backend/optimize_for